src/action/FeedFoodToPerson.cpp

```diff
@@ -261,12 +269,14 @@ void feedFoodToPerson(const std::shared_ptr<Perception> &perception,
   if (moveIFOSuccess) {
     // ===== EATING =====
     ROS_WARN("Human is eating");
-    talk("Ready to eat!");
+    if (useSound) 
+      talk("Ready to eat!");
```

Comment:
Does ada_feeding use clang_format?
Because I'm pretty sure unbracketed ifs aren't allowed.
This can be pushed as-is right now if there is no clang-format call, and we should add one soon.
If there is a clang-format call, let's do that first.

Reply:
We don't have clang_format yet. I added an issue for it https://github.com/personalrobotics/ada_feeding/issues/10